examples/cypress/webpack.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+	module: {
+		rules: [
+			{
+				test: /\.(jsx|js)$/,
+				exclude: /(node_modules)/,
+				use: {
+					loader: 'babel-loader',
+					options: {
+						presets: ['@babel/preset-env', '@babel/preset-react'],
+					},
+				},
+			},
+		],
+	},
+	resolve: {
+		extensions: ['.js', '.jsx'],
+	},
+};
```

Comment:
Not sure if the indenting on these files is really large or if it's just github..